frontends/main/src/app-pages/Articles/ArticleEditPage.tsx

```diff
@@ -0,0 +1,134 @@
+"use client"
+import React, { useEffect } from "react"
+import { Permission } from "api/hooks/user"
+import { CKEditorClient } from "ol-ckeditor"
+import { useRouter } from "next-nprogress-bar"
+import { useArticleDetail, useArticlePartialUpdate } from "api/hooks/articles"
+import { Button, Input, Alert } from "@mitodl/smoot-design"
+import RestrictedRoute from "@/components/RestrictedRoute/RestrictedRoute"
+import { Container, Typography, styled, LoadingSpinner } from "ol-components"
+import { notFound } from "next/navigation"
+
+const SaveButton = styled.div({
+  textAlign: "right",
+  margin: "10px",
+})
+
+const ClientContainer = styled.div({
+  width: "100%",
+  margin: "10px 0",
+})
+
+const TitleInput = styled(Input)({
+  width: "100%",
+  margin: "10px 0",
+})
+
+const ArticleEditPage = ({ articleId }: { articleId: string }) => {
+  const router = useRouter()
+
+  const id = Number(articleId)
+  const { data: article, isLoading } = useArticleDetail(id)
+
+  const [title, setTitle] = React.useState<string>("")
+  const [editorContent, setEditorContent] = React.useState<string>("")
+  const [editorKey] = React.useState(0)
+  const [alertText, setAlertText] = React.useState("")
+  const [severity, setSeverity] = React.useState<"success" | "error">("success")
```

Comment:
Let's not use state for this... check `articleUpdate.isError`. (I don't think clearing the error upon title change, body change, is necessary. It's an edge case anyway and will be cleared on a successful submission. If you want to clear it, you can do `articleUpdate.reset()`. 

```ts
const articleUpdate = useArticlePartialUpdate()
```